git_theta/lsh/e2lsh.py

```diff
@@ -0,0 +1,89 @@
+"""Classes for computing Euclidean locality-sensitive hashes"""
+
+import numpy as np
+import numba as nb
+import os
+
+from git_theta.lsh import HashFamily, RandomnessPool
+from git_theta.lsh.types import Signature, Parameter
+from git_theta.utils import EnvVarConstants
+
+
+class E2LSH(HashFamily):
```

Comment:
Changed the name to `EuclideanLSH`. I think I see where this gets confusing because when using banding they refer to the banding step as the "LSH" right? I'm fine with either name. Left it as `EuclideanLSH` for now but happy to change it.